sale_commission/models/account_move.py

```diff
@@ -93,6 +93,11 @@ def _compute_agent_ids(self):
                     record.move_id.partner_id
                 )
 
+    def _copy_data_extend_business_fields(self, values):
+        super()._copy_data_extend_business_fields(values)
+        if self.settlement_id and self.env.context.get("include_settlement", False):
```

Comment:
And this?

Reply:
Without this you loose the settlement_line_id. The only way I found without breaking the old logic was this.

Reply:
Then please put a docstring specifying why this is needed and from where this is called. If not, later on the future we won't remember why is this here, and we will remove it.